config/initializers/health_checks.rb

```diff
@@ -0,0 +1,41 @@
+class DelayedJobNeverRanCheck < OkComputer::Check
+  attr_reader :threshold
+
+  def initialize(threshold)
+    @threshold = threshold
+  end
+
+  def check
+    never_ran = Delayed::Job.where('run_at <= ?', 5.minutes.ago).count
```

Comment:
The `sleep_delay` appears to be 5 seconds by default. We'd need to find out how low the value is to get the minimum allowed time to have passed. Do you have a suggestion? Otherwise, I'd rather leave it at a fixed interval or make it overridable by the user.